### PR TITLE
apps wc & sc: default to object storage disabled

### LINKED ISSUE #361: [1] Default to object storage disabled
**Is your feature request related to a problem? Please describe.**

To improve DX (developer experience), it would be great to make it quicker to set up Compliant Kubernetes for quick issues. Currently, S3 buckets are enabled by default, which requires a developer to worry about things like creating S3 buckets, creating S3 bucket access credentials, etc. It would be great to disable this *by default* and enable it only when needed.

For QA and production, we now have the [backup dashboards](https://compliantkubernetes.io/ciso-guide/backup/) and the [QA checklist](https://compliantkubernetes.io/operator-manual/qa/), which should allow us to not forget about testing and enabling this feature when required.

**Definition of done:**

* S3 is disabled by default.
* Documentation exists on how to enable and verify it.
* All backups (InfluxDB, Harbor, etc.) which depend on S3 are disabled by default.
* Harbor (which is the biggest consumer of S3) is either using PVCs for persistence or is disabled by default.
* The ["at the minimum you should configure"](https://compliantkubernetes.io/operator-manual/exoscale/#configure-the-apps) sections in documentation are way shorter.

### PR DESCRIPTION
**Pull request by robinelastisys**
**What this PR does / why we need it**:
Object storage gets disabled by default to fasten a developers creation of a cluster for test-purposes.

**Which issue this PR fixes** 
fixes #361 

**How to enable ObjectStorage?**
Before or after https://github.com/elastisys/compliantkubernetes-apps quickstart 5-6:

sc-config:
Set objectStorage:type: s3
Set harbor:persistence:type: objectStorage
Set elasticsearch:snapshot:enabled: true
Set elasticsearch:fluentd:enabled: true
Set influxDB:backup:enabled: true
Set ingressNginx:velero:enabled: true

wc-config:
Set objectStorage:type: s3
Set ingressNginx:velero:enabled: true

When finished editing:
ln -sf $CK8S_CONFIG_PATH/.state/kube_config_${SERVICE_CLUSTER}.yaml $CK8S_CONFIG_PATH/.state/kube_config_sc.yaml
./bin/ck8s apply sc  # Respond "n" if you get a WARN

for CLUSTER in "${WORKLOAD_CLUSTERS[@]}"; do
    ln -sf $CK8S_CONFIG_PATH/.state/kube_config_${CLUSTER}.yaml $CK8S_CONFIG_PATH/.state/kube_config_wc.yaml
    ./bin/ck8s apply wc  # Respond "n" if you get a WARN
done

**How to make "at the minimum you should configure" shorter?**
The editing of ${CK8S_CONFIG_PATH}/secrets.yaml could be removed. Other than that, It could be left as it is as https://github.com/elastisys/compliantkubernetes-apps, quickstart 5-6 isn't needed anymore when ObjectStorage is disabled by default? The "shortening" is that quickstart 5-6 isn't a must anymore.

**Checklist:**

- [x] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits

**Definition of done:**
- [x] S3 is disabled by default.
- [x] Documentation exists on how to enable and verify it.
- [x] All backups (InfluxDB, Harbor, etc.) which depend on S3 are disabled by default.
- [x] Harbor (which is the biggest consumer of S3) is either using PVCs for persistence or is disabled by default.
- [x] The "at the minimum you should configure" sections in documentation are way shorter.

### CODE REVIEW

File: config/config/sc-config.yaml [hunk 1]
```diff
@@ -67,7 +67,7 @@ user:
       enabled: false
 
 harbor:
-  enabled: true
+  enabled: false
```
Comment (viktor-f) [inline config/config/sc-config.yaml:70]: Harbor should still be enabled. But disable Harbor backups on line 154: https://github.com/elastisys/compliantkubernetes-apps/blob/25bd108d6832661522b4d84537eeecd11c109aca/config/config/sc-config.yaml#L154
```suggestion
  enabled: true
```

File: WIP-CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 ### Changed
 
 - The Service Cluster Prometheus now alerts based on Falco metrics. These alerts are sent to Alertmanager as usual so they now have the same flow as all other alerts. This is in addition to the "Falco specific alerting" through Falco sidekick.
+- Default to object storage disabled.
```
Comment (viktor-f) [inline WIP-CHANGELOG.md:7]:
```suggestion
- Default to object storage disabled for the dev flavor.
```

**Review by viktor-f — CHANGES_REQUESTED** [review]
A good start. :slightly_smiling_face: 

Found one error in the harbor config.

Also object storage should only be disabled by default for the dev flavor. So either move these changes to the dev config files (e.g. `config/config/flavors/dev-sc.yaml`) or undo these changes in the prod flavor config files (e.g. `config/config/flavors/prod-sc.yaml`).

**Review by OlleLarsson — COMMENTED** [review]
Looks good, but I agree with Viktor's comments that these changes should be in the dev flavor.

**Comment by robinelastisys** [comment]
Corrections have been made! :+1:

File: config/config/flavors/dev-wc.yaml [hunk 3]
```diff
@@ -2,6 +2,12 @@ global:
   issuer: letsencrypt-staging
   verifyTls: false
 
+objectStorage:
+  type: none
+
+velero:
+  enabled: true
```
Comment (viktor-f) [inline config/config/flavors/dev-wc.yaml:9]:
```suggestion
  enabled: false
```

**Review by viktor-f — CHANGES_REQUESTED** [review]
One miss, but otherwise looks good.
The merge conflict should be easy to fix. If you are uncertain how I can show you via slack :slightly_smiling_face:

**Review by viktor-f — APPROVED** [review]

**Review by OlleLarsson — APPROVED** [review]

**Pull request merged**